### PR TITLE
Fix Preact doc

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [x] Improvement (make a documentation even better)
- [x] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

### CODE REVIEW

**Review by eduardoformiga — APPROVED** [review]

File: docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx [hunk 1]
```diff
@@ -18,7 +18,9 @@ Make sure the `@faststore/cli` package from your project is updated to the lates
 ## Instructions
 
 1. Open your FastStore project code.
-2. Go to `faststore.config.js` file and add the `preact` flag in the `experimental` section:
+2. Go to `discovery.config.js` file and add the `preact` flag in the `experimental` section:
+
+> The `faststore.config.js` file is the legacy filename for `discovery.config.js`. If you still using the legacy name, please refer to [this release note]() to update it.
```
Comment (hellofanny) [inline docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx:23]: @Mariana-Caetano we don't have a release note on the portal related to this issue yet, right?
We have this one in GH: https://github.com/vtex/faststore/releases/tag/v3.0.123

Comment (Mariana-Caetano) [reply docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx:23]: @hellofanny not yet, I've created this draft release: https://docs.google.com/document/d/18yZpW9Kf2QqU3EeNxDdPCBFAjhfLv-TSp83ezl556hg/edit?tab=t.0

Feel free to comment on this, I also left some questions on there.

Comment (Mariana-Caetano) [reply docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx:23]: @hellofanny for now, I'll add the release note from GitHub, but once we have the draft ready I'll update the doc.

Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx:23]:
```suggestion
> The `faststore.config.js` file is the legacy filename for `discovery.config.js`. If you still using the legacy name, please refer to [this release note](https://github.com/vtex/faststore/releases/tag/v3.0.123) to update it.
```

File: docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx [hunk 2]
```diff
@@ -18,7 +18,9 @@ Make sure the `@faststore/cli` package from your project is updated to the lates
 ## Instructions
 
 1. Open your FastStore project code.
-2. Go to `faststore.config.js` file and add the `preact` flag in the `experimental` section:
+2. Go to `discovery.config.js` file and add the `preact` flag in the `experimental` section:
+
+> The `faststore.config.js` file is the legacy filename for `discovery.config.js`. If you still using the legacy name, please refer to [this release note](https://github.com/vtex/faststore/releases/tag/v3.0.123) to update it.
```
Comment (Mariana-Caetano) [inline docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx:23]:
```suggestion
> ⚠️ The `faststore.config.js` file is the legacy filename for `discovery.config.js`. If you still using the legacy name, please refer to [this release note](https://github.com/vtex/faststore/releases/tag/v3.0.123) to update it.
```

File: docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx (hunk 1 above)
Comment (hellofanny) [reply docs/faststore/docs/managing-performance/improving-store-performance-with-preact.mdx]: Okay!! Thank you!

**Pull request merged**